src/main/java/de/hysky/skyblocker/config/configs/UIAndVisualsConfig.java

```diff
@@ -318,6 +322,16 @@ public static class Waypoints {
         public Waypoint.Type waypointType = Waypoint.Type.WAYPOINT;
     }
 
+	public static class GyroKineticWandOverlay {
+
+		@SerialEntry
+		public GyroOverlay.Mode gyroOverlayMode = GyroOverlay.Mode.OFF;
+
+
+		@SerialEntry
+		public Color gyroOverlayColor = new Color(0x7F761594, true);
+	}
```

Comment:
This is not a review yet but please make the config fields and classes order match. For example your category config entry is after `healthBars.emptyBarColor`, but your field is after `teleportOverlay`, and this is before `teleportOverlay`. Please fix their order for merge conflict purposes.